frontend/src/pages/LearningCourse/Exercise/Exercise.tsx

```diff
@@ -0,0 +1,35 @@
+import { FC } from 'react';
+
+import { IExercise } from 'types/course';
+
+import {
+  CourseInfoWrapper,
+  CourseInfoContent,
+  CourseInfoTitle,
+  CourseInfoText,
+  CodeBlock,
+} from './styled';
+
+interface IExerciseProps {
+  courseExersice: IExercise;
+  isCourseInfoOpen: boolean;
+}
```

Comment:
please move to types folder with index.ts inside it

Reply:
i think that it's odd